flow-typed/content.js

```diff
@@ -24,7 +24,7 @@ declare type WatchHistory = {
 declare type PlayingUri = {
   uri?: ?string,
   primaryUri?: string,
```

Comment:
- [ ] Embeds in Post/Comment/Livestream should not autoplay-next (it doesn't in production).

Reply:
I still see it happening in d9135b3b, but not reliably.  I think it's due to Lighthouse not working at the moment (https://odysee-workspace.slack.com/archives/C02FH1UGKGR/p1657891988222109).  Let's wait a bit until lighthouse is back -- if still can't reproduce, I'll grab the steps.

- [ ] By the way, if I disable Floating Player in the settings, the Post embeds play in the background (can only hear audio).